schema/resolvers.js

```diff
@@ -180,17 +189,39 @@ const Resolvers = {
   },
 
   Answer: {
-    __resolveType: ({ type }) =>
-      includes(["Checkbox", "Radio"], type)
-        ? "MultipleChoiceAnswer"
-        : "BasicAnswer"
+    __resolveType: ({ type }) => {
+      if (includes(["Checkbox", "Radio"], type)) {
+        return "MultipleChoiceAnswer";
+      } else if (includes(["DateRange"], type)) {
+        return "CompositeAnswer";
+      } else {
+        return "BasicAnswer";
+      }
+    }
   },
 
   BasicAnswer: {
     page: (answer, args, ctx) =>
       ctx.repositories.QuestionPage.get(answer.questionPageId)
   },
 
+  CompositeAnswer: {
+    childAnswers: answer => {
+      const firstAnswer = omit(answer, "secondaryLabel");
+      const secondAnswer = omit(answer, "label");
+      assign(firstAnswer, { id: `${answer.id}from` });
+      assign(secondAnswer, { id: `${answer.id}to` });
+      return [
+        firstAnswer,
+        mapKeys(secondAnswer, (value, key) => {
```

Comment:
Do we need to map keys here? Can't we just set label in the `secondAnswer` in the assign?